rbx_dom_lua/src/EncodedValue.lua

```diff
@@ -265,6 +265,22 @@ types = {
 		toPod = identity,
 	},
 
+	MaterialColors = {
+		fromPod = function(pod: { [string]: { number } })
+			local real = {}
+			for name, color in pod do
+				real[Enum.Material[name]] = Color3.fromRGB(color[1], color[2], color[3])
+			end
+			return real
+		end,
+		toPod = function(roblox: { [Enum.Material]: Color3 })
+			local pod = {}
+			for material, color in roblox do
+				pod[material.Name] = { math.floor(color.R * 255), math.floor(color.G * 255), math.floor(color.B * 255) }
```

Comment:
I think we should use round instead of floor here for better accuracy, similar to the Color3 to Color3uint8 conversion in rust
https://github.com/rojo-rbx/rbx-dom/blob/f899fbad5c0f7326cc852039e42ca926c7281b73/rbx_types/src/basic_types.rs#L409-L417

Reply:
For all values in the range `[0, 255]` there's no difference, so I'm not sure it's necessary, but we may as well. I've also clamped the individual components to be in `[0, 1]` since that's something we should probably worry about.